features/smoke_tests.feature

```diff
@@ -0,0 +1,177 @@
+Feature: Smoke tests
```

Comment:
This looks great in concept, but can I request you change the file and feature name to "Barebone" tests to fit in with the wider nomenclature?